lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Pure.hs

```diff
@@ -50,6 +55,9 @@ import Data.Foldable
 import Data.List.NonEmpty
     ( NonEmpty
     )
+import Data.Map
```

Comment:
my suggestion: use `Data.Map.Strict`

Reply:
Is it needed when importing the type `Map` rather than the functions? But maybe the `Semigroup` instance… 🤔